media-libs/openexr/files/openexr-2.3.0-fix-build-system.patch

```diff
@@ -0,0 +1,314 @@
+diff --git a/IlmImf/Makefile.am b/IlmImf/Makefile.am
+index 82efdb0..3f597e8 100644
+--- a/IlmImf/Makefile.am
++++ b/IlmImf/Makefile.am
+@@ -93,7 +93,7 @@ libIlmImf_la_SOURCES = ImfForward.h ImfAttribute.cpp ImfBoxAttribute.cpp ImfCRgb
+ 	               ImfSystemSpecific.cpp ImfZip.h ImfZip.cpp
+ 
+ 
+-libIlmImf_la_LDFLAGS = @ILMBASE_LDFLAGS@ -version-info @LIBTOOL_VERSION@ \
```

Comment:
Can we please just set `ILMBASE_LDFLAGS` and `ILMBASE_CXXFLAGS` to some sane (or emtpy) value instead of removing it everywhere? This makes the patch huge.